judgels-backends/sandalphon/sandalphon-api/src/main/java/judgels/sandalphon/api/submission/BundleSubmission.java

```diff
@@ -0,0 +1,20 @@
+package judgels.sandalphon.api.submission;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.time.Instant;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableBundleSubmission.class)
+public interface BundleSubmission {
```

Comment:
I was confused for a while by this. To prevent confusion, what if we rename this to `BundleItemSubmission` ?

Reply:
Renamed to `BundleItemSubmission`